hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormat.java

```diff
@@ -48,6 +49,8 @@
    */
   int currentVersion = 1;
 
+  String UNKNOWN_WRITE_TOKEN = "1-0-1";
```

Comment:
why does `1-0-1` make a good UNKNOWN_WRITE_TOKEN?

Reply:
Discussed f2f. It has to be numeric. this is used  as default write token when appending  in non-spark context environment (e:g : archiving)